pkg/build/builder/docker.go

```diff
@@ -335,6 +414,7 @@ func (d *DockerBuilder) dockerBuild(dir string, tag string) error {
 	}
 
 	opts := docker.BuildImageOptions{
+		Context:             ctx,
```

Comment:
what's intended to go in here?

Reply:
The newer go-dockerclient package includes a field for passing a `context.Context` to its build API, so we pass down the value that DockerBuilder.Build() has been allocating.  The buildah-based code path needs one to hand it to the image library if it needs to pull an image, so we go ahead and use it.